docs/dev_guide/STS Server.md

```
@@ -0,0 +1,967 @@
+- Default session duration - 1 hour.
+- Min session duration - 15 minutes.
+- Max session duration - 12 hours.
+
+[AWS Assume Role API](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)
+
+---
+
+## Troubleshooting
+- [Common STS Error Codes](#common-sts-error-codes)
+- [Enable Debug Logging](#enable-debug-logging)
+- [Key debug Log Patterns](#key-debug-log-patterns-to-search-for)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)